actions/summary-build/action.yml

```diff
@@ -12,18 +12,15 @@ inputs:
   whl_status:
     description: status from build step
     required: true
+  run_id:
+    description: github run id
+    required: false
 
 runs:
   using: composite
   steps:
     - run: |
-        if [ ${{ inputs.whl_status }} -eq 0 ]; then
-            # green check
-            WHL_EMOJI=$(echo -e "\xE2\x9C\x85")
-        else
-            # red x
-            WHL_EMOJI=$(echo -e "\xE2\x9D\x8C")
-        fi
+        WHL_EMOJI=`${{ github.action_path }}/../../scripts/step-status.sh ${{ inputs.whl_status }}`
```

Comment:
i see the encapsulation here, which i think is okay. the bulk of the action predates "nm-actions" and our use of it.